native-schema-registry/c/include/glue_schema_registry_schema.h

```diff
@@ -17,13 +17,17 @@ typedef struct glue_schema_registry_schema {
     //Data format name, JSON, AVRO, PROTOBUF as String
     char * data_format;
 
+    //Additional Schema Info needed, such as Protobuf message descriptor full name
```

Comment:
was not the idea to make it generic so we can store different information for different schema if needed? Since we are storing protobuf descriptor full name here, it might be a little bit confusing if we use `record_name` or `schema_type_name` (at least I am having trouble to make connections here)